server/main.go

```diff
@@ -41,24 +41,49 @@ var (
 	httpPort      = flag.Int("http_port", 8080, "HTTP serving port.")
 	schemaRootDir = flag.String("schema_root_dir", "schema/alpha", "Root directory for the predefined schemas.")
 
+	mlmdDBType					 = flag.String("mlmd_db_type", "in-memory", "Database type to use when creating MLMD instance. Supported options: mysql, in-memory, sqlite")
```

Comment:
Can you make "mysql" as the default value? Since we may have some command/config, which is using mysql, not aware of this new flag?

Reply:
Done.